compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -2985,12 +2985,10 @@ void OMR::X86::CodeGenerator::apply32BitLoadLabelRelativeRelocation(TR::Instruct
    {
    TR::Instruction *movInstructionX86   = movInstruction;
 
-
    TR_ASSERT(movInstructionX86->getOpCodeValue() == MOV4RegImm4,"wrong load instruction used for apply32BitLoadLabelRelativeRelocation\n");
 
    uint8_t *cursor = movInstructionX86->getBinaryEncoding();
-   cursor += movInstructionX86->rexPrefixLength();
-   cursor += movInstructionX86->getOpCode().getOpCodeLength();
+   cursor += movInstructionX86->getOpCode().length(movInstructionX86->rexBits()) + movInstructionX86->rexRepeatCount();
```

Comment:
Ok can you please comment rexRepeatCount to describe when it is non-zero and how the number returned from there relates to required and non-required REX prefixes because it doesn't seem to be completely obvious to me so could confuse others.